frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -793,6 +826,29 @@ pub mod pallet {
 			<MinimumUntrustedScore<T>>::set(maybe_next_score);
 			Ok(())
 		}
+
+		/// Set a solution in the queue, to be handed out to the client of this pallet in the next
+		/// call to `ElectionProvider::elect`.
+		///
+		/// This can only be set by `T::ForceOrigin`, and only when the phase is `Emergency`.
+		///
+		/// The solution is not checked for any feasibility and is assumed to be trustworthy, as any
+		/// feasibility check itself can in principle cause the election process to fail (due to
+		/// memory/weight constrains).
+		#[pallet::weight(T::DbWeight::get().reads_writes(1, 1))]
+		fn set_emergency_election_result(
+			origin: OriginFor<T>,
+			solution: ReadySolution<T::AccountId>,
+		) -> DispatchResult {
+			T::ForceOrigin::ensure_origin(origin)?;
+			ensure!(Self::current_phase().is_emergency(), <Error<T>>::CallNotAllowed);
+
+			// Note: we don't `rotate_round` at this point; the next call to
+			// `ElectionProvider::elect` will not succeed and take care of that.
```

Comment:
```suggestion
			// `ElectionProvider::elect` will succeed and take care of that.
```

Surely the round only rotates when the `elect` call succeeds.

Reply:
Should it then be 
```
the next _successful_ call to `ElectionProvider::elect` will take care of that.
```
?

Reply:
The next call to `elect` will be successful because there is a queued solution, I think the current wording fixed by coriolinus makes sense